app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -112,7 +112,7 @@ type CartAddress {
     telephone: String
     address_type: AdressTypeEnum
     selected_shipping_method: CheckoutShippingMethod
-    available_shipping_methods: [CheckoutShippingMethod]
+    available_shipping_methods: [CheckoutAvailableShippingMethod]
```

Comment:
We can split logic between different resolvers
Like `addresses: [CartAddress]! @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\CartAddresses")`
It will simplify our code and relations